gmm_client/src/client_requests.erl

```diff
@@ -0,0 +1,36 @@
+%%%-------------------------------------------------------------------
+%%% @author Piotr Świderski
+%%% @copyright (C) 2021, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 10. maj 2021 16:54
+%%%-------------------------------------------------------------------
+-module(client_requests).
+-author("Piotr Świderski").
+
+
+%% API
+-export([post_request/1, get_delete_request/2,
+         get_simple_request_body/1]).
+
+request(Method, Url, Headers, Payload, Options )->
+  {ok,_,_,_} = hackney:request(Method, Url, Headers, Payload, Options).
+
+simple_request(Method, Url) -> request(Method, Url, [], <<>>, []).
+
+get_simple_request_body(Url)->
```

Comment:
czy nie dałoby rady połączyć request, simple_request i get_simple_request_body w jedną funkcję?

Reply:
to zależy od tego czy funkcja request() może się przydać w przyszłości wywołana z innymi argumentami, a wydaje mi się, że tak, więc na razie bym zostawił

Reply:
no dokładnie, tutaj wszystko jest "krojone" pod to co Paweł dostarcza